test/qpy_compat/process_version.sh

```diff
@@ -34,4 +34,7 @@ else
 fi
 echo "Loading qpy files from $version with dev qiskit-terra"
 ../qiskit_venv/bin/python ../test_qpy.py load --version=$version
+if [[ ! -d qpy_$version ]] ; then
+    rm -rf ./$version
+fi
```

Comment:
I think the test is wrong here, in the senses that we're _in_ the `qpy_$version` directory at this point, it always exists by this line and it only contains the generated files. The venv is in the directory above where we're currently at, and since we're using `rm -rf`, we don't really need a test to see if it exists at all.

Reply:
Oh quite right, that's what I get for a lazy copy and paste while distracted.

Reply:
Fixed in: https://github.com/Qiskit/qiskit/pull/10794/commits/775e8b90442808f049fa3bd643a9618c841b9677